api/doc/openapi.yaml

```diff
@@ -1161,6 +1161,13 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/MoveValue'
+    ModuleBundlePayload:
+      type: object
+      properties:
+        modules:
+          type: array
+          items:
+            $ref: '#/components/schemas/MoveModuleBytecodePayload'
     MoveModuleBytecodePayload:
```

Comment:
for this `MoveModuleBytecodePayload`, you need remove `type` field. and add a type field to `ModuleBundlePayload` and set example: module_bundle_payload

Reply:
also `MoveModuleBytecodePayload` is not a payload anymore, probably rename it to `MoveModuleBytecode`

Reply:
actually, I found we have `MoveModuleBytecode`, so you should just remove `MoveModuleBytecodePayload` after replaced by `ModuleBundlePayload`